### ISSUE TITLE
CameraRollAndroid is undefined

### ISSUE THREAD

**Issue by NTDNguyen123** [bamlab/rn-camera-roll#8]
I'm new to React-native and it appears that this line "const CameraRollAndroid = React.NativeModules.CameraRollAndroid;" is not working, 

When I console.log ("CameraRollAndroid is " , CameraRollAndroid);
CameraRollAndroid is undefined

I've done "npm install rn-camera-roll", and "react-native link rn-camera-roll"

My configuration:
react-native --version
react-native-cli: 0.1.10
react-native: 0.21.0

Any help in this matter would be great.

Thanks!

**Comment by Almouro** [comment]
Hi @NTDNguyen123,

First, `CameraRoll` should be available for Android in react-native 0.19 and above, so I would suggest using that instead :)

If you still want to make this module work, can you check that the `CameraRollPackage` has been added to your list of packages ?
In your `MainActivity`, you should see something like 

```
protected List<ReactPackage> getPackages() {
   return Arrays.<ReactPackage>asList(
     new MainReactPackage(),
     new CameraRollPackage()
   );
}
```

If you don't see it, then add it and also add 

```
import fr.bamlab.rncameraroll.CameraRollPackage;
```

at the top of the `MainActivity` file

**Comment by Almouro** [comment]
Hi @NTDNguyen123, I'm closing this since there has been no activity for 15 days.
Feel free to reopen if the problem persists! :)
